### PR TITLE
En 1300 user management

### PR DESCRIPTION
**Pull request by nicpuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: test/ui/dashboard/DashboardPage.test.js [hunk 1]
```diff
@@ -13,6 +14,12 @@ describe('DashboardPage', () => {
 
   it('verify if exist InternalPage', () => {
     const component = shallow(<DashboardPage />);
-    expect(component.contains(<InternalPage className="DashboardPage" />)).toEqual(true);
+    expect(component.type()).toBe(InternalPage);
+    expect(component.hasClass('DashboardPage')).toEqual(true);
```
Comment (fedelocci) [inline test/ui/dashboard/DashboardPage.test.js]: toBe

File: test/ui/dashboard/UserManagement.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import UserManagement from 'ui/dashboard/UserManagement';
+import { ROUTE_USER_ADD } from 'app-init/router';
+import { Link } from '@entando/router';
+
+const component = shallow(<UserManagement onWillMount={() => {}} users={2} groups={3} />);
+
+
+describe('UserManagement', () => {
+  it('renders without crashing', () => {
+    expect(component.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/dashboard/UserManagement.test.js]: toBe

File: test/ui/dashboard/UserManagementContainer.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  describe('mapDispatchToProps', () => {
+    let props;
+
+    beforeEach(() => {
+      props = mapDispatchToProps(jest.fn());
+    });
+
+    it('should map the correct function properties', () => {
+      expect(props.onWillMount).toBeDefined();
```
Comment (fedelocci) [inline test/ui/dashboard/UserManagementContainer.test.js]: toHaveProperty

**Comment by entando-service** [comment]
All is well

**Pull request merged**